src/utils/systemJsRendering.ts

```diff
@@ -1,12 +1,26 @@
 import Variable from '../ast/variables/Variable';
+import { RenderOptions } from './renderHelpers';
 
-export function getSystemExportStatement(exportedVariables: Variable[]): string {
-	if (exportedVariables.length === 1) {
-		return `exports('${exportedVariables[0].safeExportName ||
-			exportedVariables[0].exportName}', ${exportedVariables[0].getName()});`;
+export function getSystemExportStatement(
+	exportedVariables: Variable[],
+	options: RenderOptions
+): string {
+	const _ = options.compact ? '' : ' ';
+	if (
+		exportedVariables.length === 1 &&
+		(exportedVariables[0].safeExportName || exportedVariables[0].exportName?.length === 1)
```

Comment:
The way we have to check `safeExportName` then the list of export names was what I thought would represent the chunking - that `safeExportName` would be the unique chunk export name (guaranteed to be one unique one), while `exportName` as an array would be the public names for the chunk if it is exposed?

The logic definitely doesn't quite work though as you can probably see...

Reply:
As I wrote, `safeExportName` should just be removed everywhere.